test/gpu/native/multiGPU/PREDIFF

```diff
@@ -5,12 +5,15 @@
 num_gpus=$(awk '/Number of sublocales.*/ {print $NF;}' < $2)
 num_gpus=$((num_gpus-1))
 
+# and number of launches expected by the test
+num_launches=$(cat $1.numlaunches)
+
 rm -f $1.good
 for i in $(seq 1 $num_gpus);
 do
-  echo "      4 Kernel launcher called. (subloc $i)" >> $1.good
+  echo "$num_launches Kernel launcher called. (subloc $i)" >> $1.good
 done
 
 # filter kernel launches and count them
-grep 'Kernel launch' $2 | sort | uniq -c > $2.new
+grep 'Kernel launch' $2 | sort | uniq -c | sed "s/^[[:space:]]*//" > $2.new
```

Comment:
This method of counting kernel launches from filtering the --verbose output seems really hackey to me. I'd prefer if our tests relied less on this kind of string processing.

It's out of scope for this PR but maybe what we should do is add a counters to the runtime library keeping track of things like kernel launches and add an undocumented method to get that value. Then we just update the test with `writeln(chpl_getNumberOfKernelLaunches())` or whatever.

If you agree let me know and I can either file a "quality-of-life" sort of issue to do this or just implement it myself.

Reply:
Couldn't agree more. This is https://github.com/Cray/chapel-private/issues/2404

And it is in my secret agenda to add something similar to CommDiagnostics module to replace what we have with `--verbose`. If you want a relatively simple GPU task for a change, feel free to steal it from me :)